src/cargo/util/lints.rs

```diff
@@ -60,6 +70,33 @@ pub fn verify_lints(
     }
 }
 
+fn find_lint_or_group<'a>(
+    name: &str,
+) -> Option<(
+    &'static str,
+    &LintLevel,
+    &Option<(Edition, LintLevel)>,
+    &Option<&'static Feature>,
+)> {
```

Comment:
Thats a bit large of a tuple with non-obvious positional meaning.  Normally this would be changed into a struct  or a trait but its unclear if another struct or trait is worth it.

Reply:
Because its only called in one place, its probably not worth fixing